lib/stat_tracker.rb

```diff
@@ -142,19 +142,6 @@ def worst_team(season)
     end.first
   end
 
-# ~~~ Game Methods ~~~
```

Comment:
Did you intend to delete these two methods?

Reply:
I saw that too - they occur later in the class, so just out of order a little bit.